src/main/java/student/command/EnrollInCourseCommand.java

```diff
@@ -1,4 +1,24 @@
 package student.command;
 
-public class EnrollInCourseCommand {
+import command.Command;
+import org.example.models.tables.pojos.Course;
+import org.example.models.tables.pojos.Student;
+import student.storage.PostgreSqlStudentRepository;
+
+public class EnrollInCourseCommand implements Command {
+    private PostgreSqlStudentRepository repository;
+    private Student student;
+    private Course course;
+
+    public EnrollInCourseCommand(PostgreSqlStudentRepository repository, Student student, Course course) {
```

Comment:
`PostgreSqlStudentRepository` -> `StudentRepository`

Always try to depend on abstractions and rarely or never on concrete types.